recipes/pdal/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pdal" %}
+{% set version = "1.7.1" %}
+{% set sha256 = "b780b64e68bf68cf31530a7b5822228303c671516c8282a6c6e19c3f27fbb345" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://download.osgeo.org/{{ name }}/PDAL-{{ version }}-src.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or not py36]
+
+requirements:
+  build:
+    - cmake
+    - toolchain
+    - numpy 1.11.*
+    - gdal
+    - postgresql
+    - jsoncpp
```

Comment:
Do we need to pin these somehow?

Reply:
For clarity, these may be open questions that we need to settle at a higher level. A tight pinning of `jsoncpp` is probably good to be safe. `postgresql` can probably be more loose, but if you want to err on the side of caution until we sort it out that is probably fine. Made a separate note about `gdal`.

xref: https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/34
xref: https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/35